noether/CMakeLists.txt

```diff
@@ -62,24 +62,24 @@ catkin_package(
 
 include_directories(include ${catkin_INCLUDE_DIRS})
 
-add_executable(surface_raster_planner_server
-  src/surface_raster_planner_server.cpp
+add_executable(tool_path_planner_server
+  src/tool_path_planner_server.cpp
 )
 
-target_link_libraries(surface_raster_planner_server
+target_link_libraries(tool_path_planner_server
   ${catkin_LIBRARIES}
   ${VTK_LIBRARIES}
   noether::vtk_viewer
 )
 
-add_executable(surface_raster_planner_application
-  src/surface_raster_planner_application.cpp
-)
-target_link_libraries(surface_raster_planner_application
-  ${catkin_LIBRARIES}
-  ${VTK_LIBRARIES}
-  noether::vtk_viewer
-)
+#add_executable(surface_raster_planner_application
```

Comment:
Not sure why you're leaving these commented out

Reply:
I will take a look. I am working on the noether_examples now.

Reply:
removed.